source/dub/dub.d

```diff
@@ -701,10 +701,10 @@ class Dub {
 			mkdirRecurse(mainfile.parentPath.toNativeString());
 
 			bool regenerateMainFile = settings.force || !existsFile(mainfile);
-
+			auto escapedMainFile = mainfile.toNativeString().replace("$", "$$");
```

Comment:
https://dlang.org/phobos/std_process.html#.escapeShellFileName

Reply:
unfortunately `escapeShellFileName` is not working, I got this Error `Error: unrecognized file extension d'`